[lb + ssl] aop edits (#1872)
cloudflare/cloudflare-docs: products/ssl/src/content/origin-configuration/authenticated-origin-pull.md

# Authenticated origin pull

> Authenticated origin pulls validate that requests to your origin server are coming from the Cloudflare network.

Author: ```suggestion
Authenticated origin pulls ensure requests to your origin server come from the Cloudflare network.
```
